### PR TITLE
RKE2 cert rotation applies to correct services

### PR DESCRIPTION
**Pull request by catherineluse**
This PR addresses two errors that Max found when validating https://github.com/rancher/dashboard/issues/4485.

1. When you select **Rotate all service certificates**, the `services` in the request needs to be an empty array. I have made it so that a service is only included in the array when you want to rotate an individual service.
2. The list of services that you can choose from needs to be different than RKE1. For RKE1, Norman provided a hardcoded list of services. In this comment https://github.com/rancher/dashboard/issues/4485#issuecomment-1064411965 Max provided the list of services that should be available for RKE2, so I have hardcoded them as options here if the current cluster is RKE2.

I also tested RKE1 cert rotation to make sure I didn't break it.

I also fixed an undefined error in the code block that is used for RKE1 certificate rotation. `this.cluster.mgmt.cluster.doAction` was supposed to be `this.cluster.mgmt.doAction`.

### CODE REVIEW

File: components/dialog/RotateCertificatesDialog.vue
Comment (richard-cox) [inline components/dialog/RotateCertificatesDialog.vue:33]: provisioning clusters have a `isRke2` property. It does exactly the same thing but it's neater to use it

**Review by richard-cox — APPROVED** [review]
One minor comment on code that was moved within the component, but I don't think that needs to be fixed before merging, so LGTM

File: components/dialog/RotateCertificatesDialog.vue
Comment (catherineluse) [reply components/dialog/RotateCertificatesDialog.vue]: I replaced it with `this.cluster.isRke2`

**Pull request merged**
